ColabTool/TextPost/index.jsx

```diff
@@ -23,6 +23,8 @@ const TextPost = ({
   draftDetails,
   text,
```

Comment:
Heya @hharnisc 😄 

Just adding the `retweetComment` and `retweetCommentLinks` to the `TextPost` as it seems we can't add separate media to retweets so this is the only post-type that should be triggered. 

I was even thinking it might be useful to enforce this on buffer-web - always use `TextPost` if it's a retweet. Curious how this feels?